<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboardView</class>
 <widget class="QMainWindow" name="AdminDashboardView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="headerHLayout" stretch="0,1">
      <item>
       <widget class="QLabel" name="headerLbl">
        <property name="font">
         <font>
          <family>Berlin Sans FB</family>
          <pointsize>40</pointsize>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>Admin Dashboard</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="indent">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="headerSpacer_MH">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="middleHLayout">
      <item>
       <layout class="QVBoxLayout" name="navVLayout">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QPushButton" name="statBtn">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../WidgetsStack.qrc">
            <normaloff>:/WidgetsStack/Assets/statistics-icon.png</normaloff>:/WidgetsStack/Assets/statistics-icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="waitingListBtn">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../WidgetsStack.qrc">
            <normaloff>:/WidgetsStack/Assets/icons8-list-64.png</normaloff>:/WidgetsStack/Assets/icons8-list-64.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addAdminBtn">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../WidgetsStack.qrc">
            <normaloff>:/WidgetsStack/Assets/add-admin-icon.png</normaloff>:/WidgetsStack/Assets/add-admin-icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="logoutBtn">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../WidgetsStack.qrc">
            <normaloff>:/WidgetsStack/Assets/logout-icon.png</normaloff>:/WidgetsStack/Assets/logout-icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="usersTable">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableWidget {
alternate-background-color: rgb(102, 147, 206);
}


QHeaderView::section { 
color:white; 
background-color:#297EB7; 
}</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>70</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>170</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>National ID</string>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Full Name </string>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Gender</string>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Age</string>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Country</string>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Government</string>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Vaccine State</string>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Doses Number</string>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomHLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="searchLbl">
        <property name="font">
         <font>
          <family>Berlin Sans FB</family>
          <pointsize>25</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="nationalIdVLayout">
        <item>
         <widget class="QLineEdit" name="nationalIdInput">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
border: 2px  ;
border-radius: 10px;
padding-left: 12px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>14</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string>National ID</string>
          </property>
          <property name="cursorMoveStyle">
           <enum>Qt::LogicalMoveStyle</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nationalIdError">
          <property name="font">
           <font>
            <family>Berlin Sans FB</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(218, 0, 0)
}</string>
          </property>
          <property name="text">
           <string>Invalid National ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deleteMsg">
          <property name="font">
           <font>
            <family>Berlin Sans FB</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 170, 0);</string>
          </property>
          <property name="text">
           <string>The user(s) deleted successfully</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="searchBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../WidgetsStack.qrc">
          <normaloff>:/WidgetsStack/Assets/search-icon.png</normaloff>:/WidgetsStack/Assets/search-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../WidgetsStack.qrc">
          <normaloff>:/WidgetsStack/Assets/delete-icon.png</normaloff>:/WidgetsStack/Assets/delete-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="previousBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../WidgetsStack.qrc">
          <normaloff>:/WidgetsStack/Assets/previous-icon.png</normaloff>:/WidgetsStack/Assets/previous-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentTabLbl">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Berlin Sans FB</family>
          <pointsize>40</pointsize>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../WidgetsStack.qrc">
          <normaloff>:/WidgetsStack/Assets/next-icon - Copy.png</normaloff>:/WidgetsStack/Assets/next-icon - Copy.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../WidgetsStack.qrc"/>
 </resources>
 <connections/>
</ui>
